test/ansible-inventory/molecule/test-local/prepare.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Prepare
+- name: Prepare for Molecule Test
```

Comment:
and the ones in this file too

Reply:
However, when we are trying to follow the logs is hard to know from where the log came and has more than one ansible test with these names.

Reply:
I am closing it as solved since after our chat it shows make sense for you too.